### PR TITLE
Feat: Import HAR file

### PR DESCRIPTION
**Pull request by farhan-sauce**
**Feature**: Import HAR file

- use `react-dropzone` to import HAR file.
- implmenet Notification component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,13 @@
...
   "dependencies": {
-    "immutable": "^3.8.2",
-    "react": "^16.11.0",
+    "bootstrap": "^4.3.1",
+    "immutable": "3.8.2",
+    "prop-types": "^15.7.2",
+    "react": "^16.10.2",
+    "react-bootstrap": "^1.0.0-beta.14",
     "react-dom": "^16.11.0",
+    "react-dropzone": "^10.1.10",
```
Comment (seanenright) [inline package.json]: should we use-exact to keep consistent with web-ui practices?

File: src/state/network/reducer.js [hunk 2]
```diff
@@ -63,6 +64,20 @@ const reducer = (state = initialState, { type, payload }) => {
           .set('data', sortBy(state.get('data'), payload.key, payload.isAcs));
       });
     }
+    case types.ADD_NOTIFICATION: {
+      const existingNotification = state.get('notifications');
+      const lastId = existingNotification.size
+        ? existingNotification.last().id : 0;
+      return state.setIn(['notifications', existingNotification.size], {
```
Comment (seanenright) [inline src/state/network/reducer.js]: will this show a different alert banner for each error like in our current UI? 
if so, is it possible to combine alerts into one error banner?

File: package.json (hunk 1 above)
Comment (farhan-sauce) [reply package.json]: I agree I will change this.

File: src/state/network/reducer.js (hunk 2 above)
Comment (farhan-sauce) [reply src/state/network/reducer.js]: That's a good suggestion. 💯 

Yes, it's the different alert banner for each `error` same as our current UI, if we want to achieve as you are saying we need to categorize notification store in `info`, `success` and `error`. 
But as of now, we throw only one error when there is an issue in importingHAR file and I don't see any other use of notification for now.
I would say let's come back here once we start dispatching more notifications.

**Pull request merged**